spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/receiving-messages/filtering.adoc

```diff
@@ -27,3 +27,62 @@ public void listen(Thing thing) {
 }
 ----
 
+### Ignore empty batch when you use `batch` mode.
+Starting with version `3.3.0`, we support ignoring empty batch that result from filtering by `RecordFilterStrategy`.
+When implementing `RecordFilterStrategy`, you can configure it through `ignoreEmptyBatch()`. default is `false`, that means `KafkaListener` will be invoked even if all `ConsumerRecord` are filtered out.
```

Comment:
One sentence per line rule.
Plus it is not clear why you start a new sentence with lower case.

Reply:
I see. 
I added `+` at end of the sentence to make new line!

Reply:
No, it had to be a new line in the code, not final text. See ASCIIDoc manual: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line